File: apps/problems/UnderstandingFeatureScaling/boilerplate-full/function.py
##USER_CODE_HERE##
  
def main():
    input_file_path = '/dev/problems/two-sum/tests/inputs/##INPUT_FILE_INDEX##.txt'
    with open(input_file_path, 'r') as file:
        input_data = file.read().strip().split('\n')
    input_list = [int(x) for x in ' '.join(input_data).split()]
      
    size_features = int(input_list.pop(0))
    features = [int(x) for x in input_list[:size_features]]
      
    result = min_max_scale(features)
      
    print(result)
  
if __name__ == "__main__":
    main()
    

File: apps/problems/UnderstandingFeatureScaling/boilerplate-full/function.cpp
#include <iostream>
#include <fstream>
#include <vector>
#include <string>
#include <sstream>
#include <climits>

##USER_CODE_HERE##

int main() {
  std::ifstream file("/dev/problems/understanding-feature scaling/tests/inputs/##INPUT_FILE_INDEX##.txt");
  std::vector<std::string> lines;
  std::string line;
  while (std::getline(file, line)) lines.push_back(line);

  file.close();
  int size_features;
  std::istringstream(lines[0]) >> size_features;
  std::vector<float> features(size_features);
  if(!size_features==0) {
  	std::istringstream iss(lines[1]);
  	for (int i=0; i < size_arr; i++) iss >> arr[i];
  }
  list<float> result = min_max_scale(features);
  std::cout << result << std::endl;
  return 0;
}
